### PR TITLE
Use Examples Returning Non-Trivial Response

### PR DESCRIPTION
**Pull request by bh2smith**
I was testing out these endpoints and found it confusing that the example values return unexpected response:

```sh
curl -X POST $NODE_URL \
   --data '{"jsonrpc":"2.0","method":"eth_getTransactionByBlockNumberAndIndex","params":["0x29c", "0x0"],"id":1}'

{"jsonrpc":"2.0","id":1,"result":null}%
```
Turns out that block `668` contains no transactions: cf https://etherscan.io/block/668

So I picked an example [transaction](https://etherscan.io/tx/0xc215b9356db58ce05412439f49a842f8a3abe6c1792ff8f2c3ee425c3501023c) that actually exists. It is the transaction with the highest total fee paid (10_668 ETH) -- cf [Dune Analytics](https://dune.com/queries/3016401).

## Test Plan

Observe that:

```sh
curl -X POST $NODE_URL \
   --data '{"jsonrpc":"2.0","method":"eth_getTransactionByBlockNumberAndIndex","params":["0x9c47cf", "0x24"],"id":1}'
```
returns

```
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": {
    "blockHash": "0x06698ca6eea2b9e6223e5bce41a142796d8845e4147a74356437054e3949fb06",
    "blockNumber": "0x9c47cf",
    "chainId": "0x1",
    "from": "0xcdd6a2b9dd3e386c8cd4a7ada5cab2f1c561182d",
    "gas": "0x5208",
    "gasPrice": "0x70ce7011fc4d400",
    "hash": "0xc215b9356db58ce05412439f49a842f8a3abe6c1792ff8f2c3ee425c3501023c",
    "input": "0x",
    "nonce": "0x458",
    "r": "0x35c4421e50b5eaaee32c65327e06feaba60c31c1f8b2656734e93c4469d71c0d",
    "s": "0x24360f723767dfa26896dce8a37671585eea2bc6f6eadf4d2c56306e2b51c5b3",
    "to": "0xe87fda75fec7063c3d2bef6dc4f17c008b098c30",
    "transactionIndex": "0x24",
    "type": "0x0",
    "v": "0x25",
    "value": "0x12f939c99edab80000"
  }
}
```

### CODE REVIEW

**Review by minimalsm — APPROVED** [review]
Thanks @bh2smith!

Makes sense to me, @jmcook1186 what do you think?

File: src/content/developers/docs/apis/json-rpc/index.md [hunk 1]
```diff
@@ -1245,8 +1245,8 @@ Returns information about a transaction by block number and transaction index po
 
 ```js
 params: [
-  "0x29c", // 668
```
Comment (wackerow) [inline src/content/developers/docs/apis/json-rpc/index.md:1248]: lol 😆 That's really funny that this example had no transactions... rugged!

File: src/content/developers/docs/apis/json-rpc/index.md [hunk 2]
```diff
@@ -1257,7 +1257,7 @@ See [eth_getTransactionByHash](#eth_gettransactionbyhash)
 
 ```js
 // Request
-curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getTransactionByBlockNumberAndIndex","params":["0x29c", "0x0"],"id":1}'
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getTransactionByBlockNumberAndIndex","params":["0x29c", "0x24"],"id":1}'
```
Comment (wackerow) [inline src/content/developers/docs/apis/json-rpc/index.md:1260]: We should just update here as well:

```suggestion
curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getTransactionByBlockNumberAndIndex","params":["0x9c47cf", "0x24"],"id":1}'
```

**Review by wackerow — APPROVED** [review]
image

Can confirm this looks good, just going to update the `curl` example with the same and pull this in, and also update on non-English pages. Good catch, thanks @bh2smith

**Comment by wackerow** [comment]
@all-contributors please add @bh2smith for bug fix

**Pull request merged**